src/Test/com/theironyard/novauc/MainTest.java

```diff
@@ -0,0 +1,61 @@
+package com.theironyard.novauc;
+
+import org.junit.Test;
+
+import java.sql.PreparedStatement;
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class MainTest {
+
```

Comment:
Well Done Peter.  Probably would want to have a separate database connection for tests but no big deal.